library/src/main/java/com/datatorrent/lib/io/fs/FileSplitterInput.java

```diff
@@ -515,6 +521,9 @@ protected ScannedFileInfo createScannedFileInfo(Path parentPath, FileStatus pare
     protected static boolean skipFile(@SuppressWarnings("unused") @NotNull Path path, @NotNull Long modificationTime,
         Long lastModificationTime) throws IOException
     {
+      if(lastModificationTime == null) {
```

Comment:
The check lastModificationTime == null becomes redundant in the next line.
Why was this change necessary? 
Please revert it.